<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Metronome</class>
 <widget class="QDialog" name="Metronome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="welcomeLabel">
     <property name="styleSheet">
      <string notr="true">QLabel { font-style:italic; }</string>
     </property>
     <property name="text">
      <string>Welcome to %1.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="storageLabel">
     <property name="text">
      <string>As this is the first time the program is launched, you can choose where %1 will find the metronome server.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
    <item>
      <widget class="QLabel" name="lblExplanation1">
        <property name="text">
          <string>The metronome server is essential to reduce the networks's overal energy consumption.</string>
        </property>
        <property name="wordWrap">
          <bool>true</bool>
        </property>
      </widget>
    </item>
    <item>
      <widget class="QLabel" name="lblExplanation2">
        <property name="textInteractionFlags">
          <enum>Qt::LinksAccessibleByMouse</enum>
        </property>
        <property name="openExternalLinks">
          <bool>true</bool>
        </property>
        <property name="text">
          <string>Setting up your own metronome is preferable, however if you do NOT want to do it, pick a public one from this list:</string>
        </property>
        <property name="wordWrap">
          <bool>true</bool>
        </property>
      </widget>
    </item>
    <item>
      <widget class="QLabel" name="lblExplanation3">
        <property name="textInteractionFlags">
          <enum>Qt::TextSelectableByMouse</enum>
        </property>
        <property name="text">
          <string>bitcoinle.org/#/metronomes</string>
        </property>
        <property name="wordWrap">
          <bool>true</bool>
        </property>
      </widget>
    </item>
   <item>
    <widget class="QLabel" name="metronomeCustom">
     <property name="styleSheet">
       <string notr="true">QLabel { margin-top:15px; }</string>
     </property>
     <property name="text">
      <string>Insert metronome server connection settings:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
        <item>
          <spacer>
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
              <size>
                <width>30</width>
                <height>10</height>
              </size>
            </property>
          </spacer>
        </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
            <widget class="QLabel" name="addrLabel">
              <property name="styleSheet">
                <string notr="true">QLabel { font-style:normal; margin-right:11px; }</string>
              </property>
              <property name="text">
                <string>Address:</string>
              </property>
              <property name="wordWrap">
                <bool>true</bool>
              </property>
            </widget>
          </item>
         <item>
          <widget class="QLineEdit" name="metronomeServer"/>
         </item>
        </layout>
       </item>
       <item>
         <spacer>
           <property name="orientation">
             <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
             <size>
               <width>30</width>
               <height>10</height>
             </size>
           </property>
         </spacer>
       </item>
        <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2_1">
            <item>
              <widget class="QLabel" name="portLabel">
                <property name="styleSheet">
                  <string notr="true">QLabel { font-style:normal; padding-left: 1px; margin-right: 2px; }</string>
                </property>
                <property name="text">
                  <string>Port:         </string>
                </property>
                <property name="wordWrap">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLineEdit" name="metronomePort"/>
            </item>
          </layout>
        </item>
        <item>
          <spacer>
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
              <size>
                <width>30</width>
                <height>10</height>
              </size>
            </property>
          </spacer>
        </item>
        <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2_2">
            <item>
              <widget class="QLabel" name="usernameLabel">
                <property name="styleSheet">
                  <string notr="true">QLabel { font-style:normal; margin-right:2px; }</string>
                </property>
                <property name="text">
                  <string>Username:</string>
                </property>
                <property name="wordWrap">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLineEdit" name="metronomeUsername"/>
            </item>
          </layout>
        </item>
        <item>
          <spacer>
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
              <size>
                <width>30</width>
                <height>10</height>
              </size>
            </property>
          </spacer>
        </item>
        <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2_3">
            <item>
              <widget class="QLabel" name="passwordLabel">
                <property name="styleSheet">
                  <string notr="true">QLabel { font-style:normal; margin-right:1px; }</string>
                </property>
                <property name="text">
                  <string>Password: </string>
                </property>
                <property name="wordWrap">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLineEdit" name="metronomePassword"/>
            </item>
          </layout>
        </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="freeSpace">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="errorMessage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Intro</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
